app/views/src/screens/Playlist.js

```diff
@@ -35,70 +33,60 @@ class Playlist extends React.Component {
             <span className="sr-only">Loading...</span>
           </div>
         </div>
-      )
+      );
     }
     if (error) {
-      return <h3>{error.message}</h3>
+      return <h3>{error.message}</h3>;
     }
-    if (playlist && showComments) {
-      return (
-        <Fragment>
-          <div className="row">
-            <div className="col">
-              <div id="playlist-heading">
-                <h2>{playlist.name}</h2>
-                <img src={playlist.bucketImageURL} alt="playlist-artwork" />
-              </div>
-              <button className="btn btn-warning comment-toggle" onClick={this.toggleComments}>Show Songs</button>
-              <Likes onLike={this.like} playlist={playlist} user={this.state.user} />
-
-            </div>
-          </div>
-          <div className="row">
-            <div className="col">
-              <Comments
-                comments={playlist.comments}
-                onSubmitComment={this.submitComment}
-                onDeleteComment={this.deleteComment}
-                user={this.state.user}
-              />
-            </div>
+
+    return (
+      <div className="row">
+        <div className="col">
+          <div id="playlist-heading">
+            <h2>{playlist.name}</h2>
+            <img src={playlist.bucketImageURL} alt="playlist-artwork" />
           </div>
-        </Fragment>
-      )
-    }
-    if (playlist) {
-      return (
-        <div className="row">
-          <div className="col">
-            <div id="playlist-heading">
-              <h2>{playlist.name}</h2>
-              <img src={playlist.bucketImageURL} alt="playlist-artwork" />
-            </div>
-            <button className="btn btn-warning comment-toggle" onClick={this.toggleComments}>Comments ({playlist.comments.length})</button>
-            <Likes onLike={this.like} playlist={playlist} user={this.state.user} />
+          <button className="btn btn-warning comment-toggle" onClick={this.toggleComments}>
+            {showComments ? 'Show songs' : `Commentssss (${playlist.comments.length})`}
+          </button>
+          <Likes onLike={this.like} playlist={playlist} user={this.state.user} />
+
+          {/* CONDITIONALY RENDER COMMENTS OR PLAYLIST */}
```

Comment:
You spelled "conditionally" wrong. Also, please do not yell at people through comments.